api/admin.proto

```diff
@@ -235,6 +235,8 @@ message GtfsRealtimeOptions {
     bool add_nyct_metadata = 4;
   }
   optional NyctAlertsOptions nyct_alerts_options = 3;
+
+  bool reassign_stop_sequences = 4;
```

Comment:
Can we add a comment to this field? (I understand the protos aren't well commented in general!) Something like:

If true, stop sequences in the GTFS realtime feed data are ignored, and alternative stop sequences are generated and assigned by Transiter. This setting is designed for buggy GTFS realtime feeds in which stop sequences (incorrectly) change between updates. In many cases Transiter is able to generate stop sequences that are correct and stable across updates.

Reply:
Added the above comment + a note about not using this for systems where a trip can call at the same stop twice.